src/abstracts/SablierV2ProxyTarget.sol

```diff
@@ -208,7 +192,7 @@ contract SablierV2ProxyTarget is
         // Calculate the sum of all of stream amounts. It is safe to use unchecked addition because one of the create
         // transactions will revert if there is overflow.
         uint256 i;
-        uint160 transferAmount;
+        uint128 transferAmount;
```

Comment:
This should have remained `uint160`

https://github.com/cantinasec/review-sablier2/issues/6

Reply:
Forgot about it